trading_backend/exchanges/binance.py

```diff
@@ -0,0 +1,46 @@
+#  Drakkar-Software trading-backend
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import trading_backend.exchanges as exchanges
+
+
+class Binance(exchanges.Exchange):
+    def __init__(self, ccxt_exchange):
+        super().__init__(ccxt_exchange)
+        self._order_custom_id = "x-bot"
+
+    @classmethod
+    def get_name(cls):
+        return 'binance'
+
+    def get_authenticated_parameters(self, params) -> dict:
+        if params is None:
+            params = {}
+        params.update({'recvWindow': 60000})
```

Comment:
Is it a test code ? (I know it's WIP, I'm just trying to understand :wink: )

Reply:
Not sure yet if we should do it but we can put this here instead of in the binance tentacle

Reply:
I don't think we should put exchange fixes into this library